ps_searchbar.tpl

```diff
@@ -23,12 +23,11 @@
 *  International Registered Trademark & Property of PrestaShop SA
 *}
 
-<div id="search_widget" data-search-controller-url="{$search_controller_url}">
-	<form method="get" action="{$search_controller_url}">
-		<input type="hidden" name="controller" value="search">
-		<input type="text" name="s" value="{$search_string}">
-		<button type="submit">
-			{l s='Search' d='Modules.Searchbar.Shop'}
-		</button>
-	</form>
+<div id="search_widget" class="search-widgets" data-search-controller-url="{$search_controller_url}">
+  <form method="get" action="{$search_controller_url}">
+    <input type="hidden" name="controller" value="search">
+    <i class="material-icons search">&#xE8B6;</i>
```

Comment:
Remember to use `aria-hidden="true"` for icons so that they are hidden from screen readers.

```suggestion
    <i class="material-icons search" aria-hidden="true">&#xE8B6;</i>
```

Also, you can change `&#xE8B6;` to the actual icon name from [material icons](https://material.io/resources/icons/) (eg. "search")